tools/isobuild/bundler.js

```diff
@@ -3246,6 +3293,10 @@ async function bundle({
   } else {
     webArchs = projectContext.platformList.getWebArchs();
   }
+  
+  if (isClientOnlyBundle) serverArch = null;
+  if (isServerOnlyBundle) webArchs = [];
```

Comment:
For the server bundle, were we previously using webArchs with all values? Not sure if it's still needed for the server. We should watch out for any breaking changes.

Later we pass webArchs when creating the server target: `targets.server = await makeServerTarget(app, webArchs);`. So will webArchs always be empty now? Not sure, maybe it's still required somewhere in the server target logic.